test/e2e/set/scyllacluster/scyllacluster_ipv6.go

```diff
@@ -0,0 +1,342 @@
+// Copyright (c) 2024 ScyllaDB.
+
+package scyllacluster
+
+import (
+	"context"
+	"strings"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/pkg/helpers"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+var _ = g.Describe("ScyllaCluster IPv6", g.Label(framework.IPv6LabelName), func() {
+	f := framework.NewFramework("scyllacluster")
+
+	g.It("should create IPv6-preferred cluster successfully", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		sc := f.GetDefaultScyllaCluster()
+		sc.Spec.Datacenter.Racks[0].Members = 1
```

Comment:
Shouldn't we create a multi-node cluster so that we verify inter-node communication works as expected? Applies to all test cases.